pkgs/development/python-modules/black/default.nix

```
@@ -20,13 +20,13 @@
 
 buildPythonPackage rec {
   pname = "black";
-  version = "21.9b0";
+  version = "21.10b0";
 
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-feTPx+trcQ3jJXEtQBJWiRAdIdJSg+7X6ZmHIs8Q65E=";
+    sha256 = "sha256:0cqg0f32hjvyjj0hg17b65qkzcirys0nvrfsygjmycif14lj55d9";
```

```suggestion
    sha256 = "sha256-qZUiKQkuMl/l89rlbYH2ObI/cTHrhAeBlH5LKIYDDzM=";
```

For future reference you can use

```sh
nix to-sri --type sha256 $the_hash
```

to produce an SRI hash.

---

or remove the sha256: prefix. Doesn't really matter which one you use.

---

Ah! Thank you for the heads up :smile: